org.cubocore.CoreImage.yml

```diff
@@ -0,0 +1,45 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
We need a more detailed justification for the filesystem=host permission. xdg-pictures would be a more appropriate default and have fewer security concerns.

Reply:
Updated to `xdg-pictures`